src/components/plots/FrequencyPlot.jsx

```diff
@@ -23,10 +24,10 @@ FrequencyPlot.defaultProps = {
 };
 
 FrequencyPlot.propTypes = {
-  hierarchy: PropTypes.object.isRequired,
+  hierarchy: PropTypes.array.isRequired,
   properties: PropTypes.object.isRequired,
   config: PropTypes.object.isRequired,
-  actions: PropTypes.object,
+  actions: PropTypes.bool,
```

Comment:
Actions can actually be boolean or object : `https://github.com/vega/vega-embed`. Maybe you can use `PropTypes.oneOfType`?